mods/cnc/rules/aircraft.yaml

```diff
@@ -2,6 +2,8 @@ TRAN:
 	Inherits: ^Helicopter
 	Valued:
 		Cost: 750
+    GivesExperience:
+        Experience: 750
```

Comment:
Nope, you seem to have handled that in the GainsExperience trait instead. What's the motivation for all these additions?